README.md

```diff
@@ -427,6 +427,7 @@ Table of Contents
   * [resmush.it](https://resmush.it) — reSmush.it is a FREE API that provides image optimization. reSmush.it has been implemented on the most common CMS such as Wordpress, Drupal or Magento. reSmush.it is the most used image optimization API with more than 7 billions images already treated, and is still Free of charge.
   * [insight.sensiolabs.com](https://insight.sensiolabs.com/) — Code Quality for PHP/Symfony projects, free for Open Source
   * [lgtm.com](https://lgtm.com) — Continuous security analysis for Java, Python, JavaScript, TypeScript, C#, C and C++, free for Open Source
+  * [Moderne.io](https://app.moderne.io) — Automatic source code refactoring. Moderne offers framework migrations, code analysis with remediation, and unrivaled code transformation at scale, so developers can spend their time building new things instead of maintaining the old. Free for Open Source.
```

Comment:
I can't see anything anywhere on the site about it being free for open source

Reply:
Hi @ripienaar,
Thanks for pointing that out; everything is defaulted to free for open source.  I have added a statement to clarify so if you follow the link you should see this now:
![image](https://user-images.githubusercontent.com/4691147/150008218-98f8fa0f-10f2-41a0-b276-998493bcd887.png)

🍻 Cheers